config/index.js

```diff
@@ -555,6 +555,12 @@ var conf = convict({
         doc: 'Temporary extra parameter to prevent request recursion',
         default: false
       }
+    },
+    clientIds: {
+      doc: 'Mappings from client id to service name: [ [ "id1", "name-1" ], [ "id2", "name-2" ] ]',
```

Comment:
How would I define this in an environment variable?

I can't do `id1,name-1,id2,name-2` because that'd just be a plain array

Reply:
I was kind of assuming with all the square brackets defined in the environment variable? Or does it not work like that?

I'm happy to define it however is most appropriate from your perspective tbh, what do you think would be best? Should I just put `format: Object` and leave it at that?

Reply:
I wish it supported the `[]` syntax! Sadly it only supports [splitting on ,](https://github.com/mozilla/node-convict/blob/master/lib/convict.js#L414).

Maybe `format: Object` is best, because then I can actually use `{}`